### PR TITLE
SAN-3325 Added custom templates

### PR DESCRIPTION
**Pull request by Myztiq**
- [ ] @thejsj

### CODE REVIEW

File: views/pages/unresponsive.jade [hunk 1]
```diff
@@ -2,7 +2,7 @@ extends ../index.jade
 block content
   h1.h1 Oops!
   p.p We can’t reach the container for
-    strong.strong  #{branchName}
+    strong.strong  #{instanceName}
```
Comment (taylordolan) [inline views/pages/unresponsive.jade]: When do we show the contents of this file? How is it different than `unknown.jade`?

Comment (Myztiq) [inline views/pages/unresponsive.jade]: Navi controls that. I'm unsure.

**Comment by taylordolan** [comment]
I added links where needed but @Myztiq I need you to make them work :)

File: views/pages/unresponsive.jade (hunk 1 above)
Comment (taylordolan) [inline views/pages/unresponsive.jade]: Okay, I just want to make sure that we're not telling them the container is running if it's not. Also this file matches `running.jade` now. Do with that what you will.

**Comment by taylordolan** [comment]
It hid my comment on `unresponsive.jade` because the diff was outdated. I just want to make sure that we're not telling them the container is running if it's not. Also `unresponsive.jade` matches `running.jade` now. Do with that what you will.

**Comment by Myztiq** [comment]
We don't have a way to deep link to opening the edit modal on a particular page. I don't think we even have support to scroll to a particular portion of the configure page anymore.

**Comment by taylordolan** [comment]
Can't we just link them to the containers view?

**Comment by Myztiq** [comment]
Yea, but it wont' really do anything useful for them... won't auto scroll to their container or anything. Still want that?

**Comment by taylordolan** [comment]
I mean can't we send them to the relevant container in the containers view, not just the containers view itself

**Comment by thejsj** [comment]
Code LGTM. Makes it a lot cleaner.

**Pull request merged**